android/src/org/kiwix/kiwixmobile/ZimFileSelectFragment.java

```diff
@@ -120,19 +120,20 @@ public void refreshFragment(){
     if (mZimFileList == null)
       return;
 
-    mZimFileList.addFooterView(progressBar);
     mZimFileList.setOnItemClickListener(this);
     mZimFileList.setOnItemLongClickListener(this);
 
     bookDao = new BookDao(KiwixDatabase.getInstance(context));
+    ArrayList<LibraryNetworkEntity.Book> books = bookDao.getBooks();
+    Collections.sort(books, new fileComparator());
 
-    mFiles = bookDao.getBooks();
-    Collections.sort(mFiles, new fileComparator());
-
-    mRescanAdapter = new RescanDataAdapter(ZimFileSelectFragment.context, 0, mFiles);
+    for (LibraryNetworkEntity.Book book : books)
```

Comment:
No I mean missing { its not essential but it makes the code a bit more readable.